### ISSUE TITLE
RCS1055 False Positive for (empty) primary constructors

### ISSUE THREAD

**Issue by hankovich** [dotnet/roslynator#1134]
**Product and Version Used**: 4.3.0

**Actual Behavior**:
```csharp
internal sealed class ChaosUpdatedMessage; // <- RCS1055 is reported here
```

**Comment by JosefPihrt** [comment]
This feature is currently in preview. Let's wait when it's official.
